examples/wallet/app/components/RestoreWalletFromMnemonic.js

```diff
@@ -0,0 +1,164 @@
+// @flow
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import typeof { setAccountFromMnemonic as SetAccountFromMnemonic } from '../actions/account';
+import { isValidMnemonic } from '../utils/mnemonic';
+
+type Props = {
+  setAccountFromMnemonic: SetAccountFromMnemonic
+};
+
+export default ({ setAccountFromMnemonic }: Props) => {
+  const checkIsValidMnemonicPhrase = function checkIsValidMnemonicPhrase() {
+    setIsMnemonicValid(isValidMnemonic(newMnemonicPhrase));
+  };
+
+  const handleSubmitMnemonic = function handleSubmitMnemonic(event) {
+    event.preventDefault();
+    if (isValidMnemonic(newMnemonicPhrase)) {
+      if (checkValidUnlockWalletPassword(password, confirmPassword)) {
+        return setAccountFromMnemonic(
+          newMnemonicPhrase,
+          newMnemonicPassword,
+          password
+        );
+      }
+    } else {
+      setIsMnemonicValid(false);
+    }
+  };
+
+  const [isMnemonicValid, setIsMnemonicValid] = useState(true);
+
+  const [newMnemonicPhrase, setNewMnemonicPhrase] = useState('');
+
+  const [newMnemonicPassword, setNewMnemonicPassword] = useState('');
+
+  const checkValidUnlockWalletPassword = function checkValidUnlockWalletPassword(
+    pass,
+    confirmation
+  ) {
+    if (!pass && !confirmation) return true;
+    if (pass !== confirmation) {
+      setArePasswordAndConfirmationEqual(false);
+      return false;
+    }
+    setArePasswordAndConfirmationEqual(true);
+
+    if (!/^[0-9a-zA-Z]{8,}$/.test(password)) {
+      setIsValidPassword(false);
+      return false;
+    }
+
+    setIsValidPassword(true);
+    return true;
+  };
+
+  const [
+    arePasswordAndConfirmationEqual,
+    setArePasswordAndConfirmationEqual
+  ] = useState(true);
+
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [isValidPassword, setIsValidPassword] = useState(true);
+
+  return (
+    <Container>
+      <Form onSubmit={handleSubmitMnemonic} className="mt-5">
+        <Form.Group>
+          <Form.Label>Wallet Seed:</Form.Label>
+          <Form.Control
+            required
+            type="text"
+            id="mnemonicPhrase"
+            name="mnemonicPhrase"
+            placeholder="Enter your secret word phrase here to restore your vault. The phrase can have 12, 15, 18, 21 or 24 words."
+            value={newMnemonicPhrase}
+            isInvalid={!isMnemonicValid}
+            onChange={event => setNewMnemonicPhrase(event.target.value)}
+            onBlur={() => checkIsValidMnemonicPhrase()}
+          />
+          <Form.Label className="text-danger" hidden={isMnemonicValid}>
```

Comment:
use [Form.Control.Feedback](https://react-bootstrap.netlify.com/components/forms/#form-control-feedback-props) for this.
Also, research the props of that component to see if you can style it by passing a prop (some have a `plaintext` prop)

If that's not possible, consider using `<span>`, which is a more idiomatic way to apply styles to inline text, without implying a semantic distinction (`<em>` is used for emphasis)

Reply:
span is used instead em